android/app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "69589465874",
+    "project_id": "cozy-flagship-app-push-prod",
+    "storage_bucket": "cozy-flagship-app-push-prod.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:69589465874:android:6723756bf1524edc7c91cb",
+        "android_client_info": {
+          "package_name": "io.cozy.flagship.mobile"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "69589465874-vgk2mu3b9q2k9nqmo3nm8t5s1l89iur7.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBnwMivfVMpmgBTeDqgeznYW52k8ddy3yo"
+        }
+      ],
+      "services": {
+        "appinvite_service": {
+          "other_platform_oauth_client": [
+            {
+              "client_id": "69589465874-vgk2mu3b9q2k9nqmo3nm8t5s1l89iur7.apps.googleusercontent.com",
+              "client_type": 3
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "configuration_version": "1"
+}
```

Comment:
ultra nitpick sorry but: `No new line at end of file`

Reply:
it seems your editor doesn't respect `.editorconfig` @zatteo

Reply:
Right! I updated my editor.